cryostat/templates/NOTES.txt

```diff
@@ -73,17 +76,17 @@
 {{ $listNum }}. {{ "Visit the " }}{{ .Chart.Name | camelcase }}{{ " application at: " }}
   ```
 {{- if .Values.core.route.enabled }}
-  echo http{{ if $.Values.core.route.tls.enabled }}s{{ end }}://$ROUTE_HOST
+  http{{ if $.Values.core.route.tls.enabled }}s{{ end }}://$ROUTE_HOST
 {{- else if .Values.core.ingress.enabled -}}
 {{- range $host := .Values.core.ingress.hosts -}}
   {{- range .paths -}}
   http{{ if $.Values.core.ingress.tls }}s{{ end }}://{{ $host.host }}{{ .path }}
   {{- end }}
 {{- end }}
 {{- else if contains "NodePort" .Values.core.service.type }}
-  echo http://$NODE_IP:$NODE_PORT
+  http://$NODE_IP:$NODE_PORT
 {{- else if contains "LoadBalancer" .Values.core.service.type }}
-  echo http://$SERVICE_IP:{{ .Values.core.service.port }}
+  http://$SERVICE_IP:{{ .Values.core.service.httpPort }}
 {{- else if contains "ClusterIP" .Values.core.service.type }}
```

Comment:
I think we might want to keep the `echo` for these since running the command in the terminal will give you the URL with environment variables expanded. That's a great catch on that service port though. Would you mind opening another PR to fix that? We should backport that in case we decide to do a 2.3.1 release.

Reply:
Thanks! Open here: https://github.com/cryostatio/cryostat-helm/pull/68